ssc/cmod_pvsamv1.cpp

```diff
@@ -1020,13 +1020,13 @@ void cm_pvsamv1::exec( ) throw (general_error)
 
 	std::vector<ssc_number_t> p_pv_clipping_forecast;
 	std::vector<ssc_number_t> p_pv_dc_forecast;
-	std::vector<ssc_number_t> p_pv_dc_use;
+	std::vector<ssc_number_t> p_pv_ac_use;
 
 	if (is_assigned("batt_pv_clipping_forecast")) {
 		p_pv_clipping_forecast = as_vector_ssc_number_t("batt_pv_clipping_forecast");
 	}
-	if (is_assigned("batt_pv_dc_forecast")) {
-		p_pv_dc_forecast = as_vector_ssc_number_t("batt_pv_dc_forecast");
+	if (is_assigned("batt_pv_ac_forecast")) {
+		p_pv_dc_forecast = as_vector_ssc_number_t("batt_pv_ac_forecast");
```

Comment:
Could you please refactor this to be named `p_pv_ac_forecast` to avoid confusion?